test/test-execute/exec-restrict-namespaces-no.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Test RestrictNamespaces=no
+
+[Service]
+RestrictNamespaces=no
+ExecStart=/bin/unshare -m -u -i -n -p -f
```

Comment:
Test passed, thanks!
I've run `rkt`-tests. Some tests failed:

> --- FAIL: TestNoNewPrivileges (37.31s) https://github.com/coreos/rkt/issues/3367
--- FAIL: TestPathsWrite (23.95s) https://github.com/coreos/rkt/issues/3245#issuecomment-253195314


But that's unrelated to this PR